src/renderer/components/subscriptions-live/subscriptions-live.js

```diff
@@ -0,0 +1,515 @@
+      try {
+        const channelInstance = await getLocalChannel(channel.id)
+
+        if (channelInstance.has_live_streams) {
+          const liveTab = await channelInstance.getLiveStreams()
+
+          if (liveTab.videos === null) {
+            this.errorChannels.push(channel)
+            return []
+          }
+
+          return parseLocalChannelVideos(liveTab.videos, channelInstance.header.author)
+        } else {
+          return []
+        }
+      } catch (err) {
+        console.error(err)
+        const errorMessage = this.$t('Local API Error (Click to copy)')
+        showToast(`${errorMessage}: ${err}`, 10000, () => {
+          copyToClipboard(err)
+        })
+        switch (failedAttempts) {
+          case 0:
+            return await this.getChannelLiveLocalRSS(channel, failedAttempts + 1)
+          case 1:
+            if (this.backendFallback) {
+              showToast(this.$t('Falling back to Invidious API'))
+              return await this.getChannelLiveInvidious(channel, failedAttempts + 1)
+            } else {
+              return []
+            }
+          case 2:
+            return await this.getChannelLiveLocalRSS(channel, failedAttempts + 1)
+          default:
+            return []
+        }
+      }
+    },
+
+    getChannelLiveLocalRSS: async function (channel, failedAttempts = 0) {
+      const playlistId = channel.id.replace('UC', 'UULV')
+      const feedUrl = `https://www.youtube.com/feeds/videos.xml?playlist_id=${playlistId}`
+
+      try {
+        const response = await fetch(feedUrl)
+
+        if (response.status === 404) {
+          return []
+        }
+
+        return await this.parseYouTubeRSSFeed(await response.text(), channel.id)
+      } catch (error) {
+        console.error(error)
+        const errorMessage = this.$t('Local API Error (Click to copy)')
+        showToast(`${errorMessage}: ${error}`, 10000, () => {
+          copyToClipboard(error)
+        })
+        switch (failedAttempts) {
+          case 0:
+            return this.getChannelLiveLocal(channel, failedAttempts + 1)
+          case 1:
+            if (this.backendFallback) {
+              showToast(this.$t('Falling back to Invidious API'))
+              return this.getChannelLiveInvidiousRSS(channel, failedAttempts + 1)
+            } else {
+              return []
+            }
+          case 2:
+            return this.getChannelLiveLocal(channel, failedAttempts + 1)
+          default:
+            return []
+        }
+      }
+    },
+
+    getChannelLiveInvidious: function (channel, failedAttempts = 0) {
+      return new Promise((resolve, reject) => {
+        const subscriptionsPayload = {
+          resource: 'channels',
+          id: channel.id,
+          subResource: 'streams',
+          params: {}
+        }
+
+        invidiousAPICall(subscriptionsPayload).then(async (result) => {
+          resolve(await Promise.all(result.videos.map((video) => {
```

Comment:
Invidious endpoint for live streams returns non video entries too
Channel tested: https://www.youtube.com/channel/UCnKbs5xffaEzzdKdPZvfhdQ
```suggestion
          // Sometimes `channel`, 'category' entries are mixed in the result
          resolve(await Promise.all(result.videos.filter(e => e.type === 'video').map((video) => {
```